gradle.properties

```diff
@@ -28,13 +28,12 @@ kapt.include.compile.classpath=false
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
-android.disableAutomaticComponentCreation=true
 # Build Features
 android.defaults.buildfeatures.buildconfig=false
 android.defaults.buildfeatures.resvalues=false
 # Library Information
 GROUP=com.deliveryhero.whetstone
-VERSION_NAME=0.5.1-SNAPSHOT
+VERSION_NAME=0.6.0-SNAPSHOT
```

Comment:
(thinking) Do we even need to release a new version if the only thing that changed is build tools? If so, should it be a patch upgrade?

cc @vsdeni @helios66 @PHELAT

Reply:
So currently I was trying to have some consistency based on the previous upgrades for different Kotlin versions. 
![Screenshot 2023-10-17 at 17 50 29](https://github.com/deliveryhero/whetstone/assets/112622447/0e490770-38f7-4975-846a-a8cf9642e662)